src/components/MeetOrganizers.jsx

```diff
@@ -0,0 +1,35 @@
+import { ABOUT_HYDPY } from '@/details';
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
```

Comment:
The border looks odd, The heading should be enough for separation. Feel free to use top padding if required

Reply:
Removed border and used padding.